### PR TITLE
@zephraph => [filterArtworksConnection] Pass proper params to connection helper

### PR DESCRIPTION
**Pull request by mzikherman**
Closes https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=64&projectKey=PLATFORM&modal=detail&selectedIssue=PLATFORM-1685

Goofed and passed the wrong set of params to the connection helper. The Relay connection args are expected, but was passing the underlying API options instead. Added a spec that passes locally.

### CODE REVIEW

File: src/schema/v2/object_identification.ts [hunk 1]
```diff
@@ -60,6 +60,10 @@ const typeNames = {
   "./filterArtworksConnection": "filterArtworksConnection",
 }
 
+const exportNames = {
```
Comment (mzikherman) [inline src/schema/v2/object_identification.ts]: I'm not sure if this was causing a bug in Emission or not (unrelated to the pagination), but noticed the `... on Node` stuff was failing (I had a corresponding spec that was passing locally but must have started failing at some point).

There was a default export that was changed to named (seems to cause weird circular dependency type things). So just added this little override. /shrug

File: src/schema/v2/filterArtworksConnection.ts [hunk 2]
```diff
@@ -392,7 +392,14 @@ const filterArtworksConnectionTypeFactory = (
       keyword_match_exact: keywordMatchExact,
       ..._options,
     }
-    const { include_artworks_by_followed_artists, aggregations } = options
+    const {
+      first,
+      last,
+      after,
+      before,
+      include_artworks_by_followed_artists,
```
Comment (zephraph) [inline src/schema/v2/filterArtworksConnection.ts]: Are you sure this is still valid? As far as I know all the snake case references were removed.

File: src/schema/v2/__tests__/filterArtworksConnection.test.js [hunk 3]
```diff
@@ -268,6 +268,64 @@ describe("filterArtworksConnection", () => {
     })
   })
 
+  describe(`Returns proper pagination information`, () => {
```
Comment (zephraph) [inline src/schema/v2/__tests__/filterArtworksConnection.test.js]: Just a heads up, this won't be run yet in CI. I should be done w/ the fix tests ticket today, but until that's finished it's ignored.

Comment (mzikherman) [reply src/schema/v2/__tests__/filterArtworksConnection.test.js]: Yep! That's how the passing `...on Node` test below in this file (which was passing locally!), started to break after a later commit changed a default export to a named one. This was just for my own sanity to run locally.

File: src/schema/v2/filterArtworksConnection.ts (hunk 2 above)
Comment (mzikherman) [reply src/schema/v2/filterArtworksConnection.ts]: Yea, if you look closer this is destructuring out of `options`, which already does the camel case (incoming args) -> snake case (for backend API calls) translation.

Comment (zephraph) [reply src/schema/v2/filterArtworksConnection.ts]: oooooohhh, right. Man, that always gets me.

**Pull request merged**